pinot-controller/src/main/resources/app/pages/TenantDetails.tsx

```diff
@@ -635,10 +774,43 @@ const TenantPageDetails = ({ match }: RouteComponentProps<Props>) => {
         <div className={classes.highlightBackground}>
           <TableToolbar name="Summary" showSearchBox={false} />
           <Grid container spacing={2} alignItems="center" className={classes.body}>
-            <Grid item xs={4}>
+            <Grid item xs={3}>
               <strong>Table Name:</strong> {tableSummary.tableName}
             </Grid>
-            <Grid item container xs={4} wrap="nowrap" spacing={1}>
+            <Grid item xs={3}>
+              <Box display="flex" alignItems="center">
+                <strong>Consuming status:</strong>
+                {loadingPauseStatus ? (
+                  <CircularProgress size={16} style={{ marginLeft: 8 }} />
+                ) : pauseStatusData ? (
+                  <Chip
+                    label={
+                      isPauseActionInProgress
```

Comment:
consuming status is only fetched for realtime tables?
if yes then this status should only be shown for realtime tables